src/main/proto/ProgramService.proto

```diff
@@ -199,10 +200,46 @@ message JoinProgramRequest {
   google.protobuf.StringValue department = 5;
 }
 
+message Invitation {
+  google.protobuf.StringValue id = 1;
+}
+
 message JoinProgramResponse {
   User user = 1;
 }
 
 message RemoveProgramRequest {
   google.protobuf.StringValue program_short_name = 1;
 }
+
+message GetJoinProgramInviteRequest {
+  google.protobuf.StringValue invite_id = 1;
+}
+
+message JoinProgramInvite {
```

Comment:
Can you format this message like the other ones in this file, removing the blank lines between every property.

Reply:
sure